packages/components/managed-data-table/src/managed-data-table.js

```diff
@@ -0,0 +1,140 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+import requiredIf from 'react-required-if';
+import Spacings from '@commercetools-uikit/spacings';
+import DataTable from '@commercetools-uikit/data-table';
+import DataTableSettings from './data-table-settings';
+
+const ManagedDataTable = (props) => {
+  const columns = props.columns.map((column) => ({
```

Comment:
Nit: we could memoize this mapping using `React.memo`

Reply:
Like
```
  const columns = React.memo(
    props.columns.map((column) => ({
      ...column,
      isTruncated: !props.displaySettings.disableDisplaySettings
        ? props.displaySettings.isWrappingText
        : column.isTruncated,
    }))
  );
```

Reply:
Sorry I meant `useMemo`. Something like this:

```js
const columns = React.useMemo(() =>
  props.columns.map((column) => ({
    ...column,
    isTruncated: !props.displaySettings.disableDisplaySettings
      ? props.displaySettings.isWrappingText
      : column.isTruncated,
  })),
  [props.columns, props.displaySettings.disableDisplaySettings, props.displaySettings.isWrappingText]
);
```

It's not super important though, I'm also fine keeping it as-is.